Gemfile

```diff
@@ -31,5 +31,6 @@ group :quality do
   gem 'flog'
   gem 'reek'
   gem 'rubocop'
+  gem 'ruby-vnu'
```

Comment:
As this is required to be able to run `rake test:all`, I think it should be in the `:test` group